### ISSUE TITLE
Unmineable Balance

### ISSUE THREAD

**Issue by sfalas01** [RainbowMiner/RainbowMiner#2360]
Unmineable does offer an APY in order to pull balances, which are currently not available in RBM [Link to API](https://www.unmineable.com/support/article/do-you-have-an-api-for-developers)
An option to specify your payout coin could be added in the Unmineable pool config in order to pull data from the API and show it in the Balances.

**Comment by RainbowMiner** [comment]
The balances should already be there. The API will be called for each coin with wallet address in pools.config.txt. I m not sure, why it's not there for you. If you like, you can send me your Unmineable wallet addresses (or one or some of them) by email and I'll check it.

**Comment by sfalas01** [comment]
> The balances should already be there. The API will be called for each coin with wallet address in pools.config.txt. I m not sure, why it's not there for you. If you like, you can send me your Unmineable wallet addresses (or one or some of them) by email and I'll check it.

Balances maybe cannot be tracked because I was passing my referral code in the wallet address. Apparently, your referral code gets appended automatically (it was not stated somewhere like zergpool - which btw it is still showing up somewhere in the web interface, not sure where - Rates?Best Pools?Not 100% sure).
So the end address was like: 'wallet_address.$WorkerName#my_referral.$WorkerName#rbm_referral

So maybe the balance module probably cannot parse the wallet address correctly?

**Comment by RainbowMiner** [comment]
Yes, that extended wallet address is the cause for the balance not showing up. And yes, the unMineable pool is missing a documentation yet. The referrals will be added to reduce the pool fee. You can use your own referral codes by putting them into the `<Coin>-Params` parameters in pools.config.txt.

For example:
`
"unMineable": {
    "LTC": "$LTC",
    "LTC-Params": "njan-ebtu",
...
  },
`

**Comment by RainbowMiner** [comment]
unMineable balances have been implemented - current master ships Balances/unMineable.ps1. Closing as obsolete for the current master (v5.0.2.1) - please open a new issue if this still occurs. Thanks for the report!
